OneApply/settings.py

```diff
@@ -126,4 +126,11 @@
 
 STATIC_ROOT = os.path.join(BASE_DIR, "staticfiles")
 
+# Setup Email Settings
+EMAIL_USE_TLS = True
+EMAIL_HOST = "smtp.gmail.com"
+EMAIL_HOST_USER = "UserVerifOneApply@gmail.com"
+EMAIL_HOST_PASSWORD = "pwd@TeamThree"
```

Comment:
is there a way to mention password somewhere else? if the password changes, then we need to keep on committing the change in the settings file.

Reply:
I was trying to figure that out. for now I left it here but any ideas would be appreciated